tools/nxfuse/.gitignore

```diff
@@ -0,0 +1,4 @@
+.config
+include/tinyara/config.h
+nxfuse
+
```

Comment:
We make nxfuse to link headers. After building, does not git say modified?
I mean don't you need to add linked path in this file?

Reply:
Files will be temporarily linked and unlinked during build of nxfuse tool.